Tests/UnitTests/Core/Vertexing/VertexingDataHelper.cpp

```diff
@@ -0,0 +1,157 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <fstream>
+#include <iterator>
+#include <regex>
+
+#include "Acts/Tests/CommonHelpers/DataDirectory.hpp"
+#include "Acts/Utilities/Definitions.hpp"
+#include "Acts/Utilities/Units.hpp"
+#include "Acts/Vertexing/Vertex.hpp"
+
+namespace Acts {
+namespace Test {
+
+using namespace Acts::UnitLiterals;
+using Covariance = BoundSymMatrix;
+
+// Create a test context
+GeometryContext geoCtx = GeometryContext();
+
+enum VertexCsvData { BeamSpotData, VerticesData, TracksData };
+
+/// @brief Helper struct to store reference vertex related information
+struct VertexInfo {
+  // The position
+  Vector3D position;
```

Comment:
Can this be `Vector4D` for future compatibility?

Reply:
The ATLAS reference data will never be 4-dim, so I would prefer keeping it like this for now. Otherwise also the covariance matrices have to be rendered 4-dim and I think it's just not necessary here (for now). If this is ever needed, we can easily change it then.